services/postprocessing/README.md

```diff
@@ -20,12 +20,25 @@ When postprocessing has been enabled, configuring any postprocessing step will r
 
 ## Postprocessing Steps
 
-As of now, `ocis` allows two different postprocessing steps to be enabled via an environment variable.
+`ocis` allows setting the order and amount of postprocessing steps via the `POSTPROCESSING_STEPS` envvar. It expects a comma seperated list of steps that should be executed. Known to the system are `virusscan` and `delay`. It is allowed to set custom steps.
 
 ### Virus Scanning
 
-To enable virus scanning as a postprocessing step after uploading a file, the environment variable  `POSTPROCESSING_VIRUSSCAN` needs to be set to ` true`. As a result, each uploaded file gets virus scanned as part of the postprocessing steps. Note that the `antivirus` service is required to be enabled and configured for this to work.
+To enable virus scanning as a postprocessing step after uploading a file, the environment variable `POSTPROCESSING_STEPS` needs to contain the word `virusscan`. As a result, each uploaded file gets virus scanned as part of the postprocessing steps. Note that the `antivirus` service is required to be enabled and configured for this to work.
 
 ### Delay
 
-Though this is for development purposes only and NOT RECOMMENDED on production systems, setting the environment variable `POSTPROCESSING_DELAY` to a duration not equal to zero will add a delay step with the configured amount of time. ocis will continue postprocessing the file after the configured delay.
+Though this is for development purposes only and NOT RECOMMENDED on production systems, setting the environment variable `POSTPROCESSING_DELAY` to a duration not equal to zero will add a delay step with the configured amount of time. ocis will continue postprocessing the file after the configured delay. Use the enviroment variable `POSTPROCESSING_STEPS` and the keyword `delay` if you have multiple postprocessing steps and want to define their order. If `POSTPROCESSING_DELAY` is set but `delay` is not contained in `POSTPROCESSING_STEPS` it will be added as last postprocessing step.
```

Comment:
```suggestion
Though this is for development purposes only and NOT RECOMMENDED on production systems, setting the environment variable `POSTPROCESSING_DELAY` to a duration not equal to zero will add a delay step with the configured amount of time. ocis will continue postprocessing the file after the configured delay. Use the enviroment variable `POSTPROCESSING_STEPS` and the keyword `delay` if you have multiple postprocessing steps and want to define their order. If `POSTPROCESSING_DELAY` is set but teh keyword `delay` is not contained in `POSTPROCESSING_STEPS`, it will be added as last postprocessing step without further notice.
```

Reply:
Not true. It will print out a message that the delay step was added so it is NOT `without further notice`

Reply:
I removed the part of the sentence, but: `It will print out a message that the delay step was added`:
1. when does this get printed
2. adding means that you will see it in the list of steps in last position?

Reply:
1. Once on service start
2. output looks like: `Added delay step to the list of postprocessing steps. NOTE: Use envvar `POSTPROCESSING_STEPS=virusscan,delay` to suppress this message and choose the order of postprocessing steps.`

Reply:
I have updated the text according your info. pls have a look. If ok, commit.